### ISSUE TITLE
Consider merging the project to KDE KRDC

### ISSUE THREAD

**Issue by paaspaas00** [F1ash/qt-virt-manager#55]
@F1ash please consider merging the project with KDE KRDC https://apps.kde.org/it/krdc/ 
You've implemented many features it really lacks, such as the ability to manage and create virtual machines etc.
The community would really benefit from this. 

See my branch https://github.com/paaspaas00/qt-virt-manager, I've tried to make it more KDE-friendly. (made some mistake including files, I've not really modified 300k LoCs)
For editing XMLs you could be using one of KDE frameworks fro syntax highlighting, or importing something like a fully blown text editor from KDE as a KPart.

And yes, I love your program!

**Comment by paaspaas00** [comment]
I've opened this in KRDC tracker https://bugs.kde.org/show_bug.cgi?id=478372
If you like the general merging idea, I'd surely contribute!

**Comment by morsik** [comment]
But this project is something totally different than KRDC...

KRDC is remote desktop viewer. Just that. And QtVirtManager is VMs configuration tool that are managed by libvirt (same as regular `virt-manager` but writte in Qt instead of GTK).

This doesn't make any sense.

**Comment by paaspaas00** [comment]
Well yes and no, KRDC allows people to manage machines, which can also be virtual machines in principle. In my opinion it would make sense for an admin point of view to have a unified tool to do all this. 
This because admins often need to do both, using VMs and connecting to remote hosts, and converging on these features would make people's life easier.
I would support the idea of having also the possibility to manage containers and systemd services from there, seamlessly, but that's my grand-vision. I think it would make for a great tool, more powerful than the sum of the two

**Comment by morsik** [comment]
@paaspaas00 KRDC does not allow to manage virutal machines at all. You can't create/start/stop them. You can't manage network cards, disk and do all stuff that you do with virtual machines. It's just remote desktop... Saying that KRDC is to manage virtual machines is biiig stretch.

It's like saying "Let's merge planes and trucks because both of them does the same - moving big stuff" - but both have totally different purpose.

**Comment by paaspaas00** [comment]
@morsik yes I know KRDC does not allow to manage VMs, said it before. Just saying having a unified tool for similar system-management stuff IMHO would be a positive thing. Of course, I don't expect everyone to agree

**Comment by SeboSauer** [comment]
Such a new tool cannot be done within KRDC but it would need to be a new app (or someone takes over this app and continues to improve it). So closed https://bugs.kde.org/show_bug.cgi?id=478372.

**Comment by SeboSauer** [comment]
fyi: A successor in active development is at https://invent.kde.org/system/karton
